### DISCUSSION TITLE
Send a soft reboot command to OCP?

### DISCUSSION THREAD

**Discussion by CodeSlinger69** [tagyoureit/nodejs-poolController#795] [Q&A] [answered]
My situation is that I have an ancient OCP, an EasyTouch 4p, likely from 2007 or so. Our humble devs here were kind enough to add support to it in this thread here: https://github.com/rstrouse/nodejs-poolController-dashPanel/discussions/66

Its on its last legs, and I need to manually hard reboot it just about every day now. This is done with a trip to the OCP, flip the breaker off, then flip it back on. I will build a Nixie eventually, I promise, but for now just trying to squeeze what life I can out of the existing equipment.

I'm wondering if there is a way to send a scheduled soft reboot signal over the RS485 interface? If I can reboot it on a daily basis from npc, then I might get a bit more buffer before I have to commit to the Nixie build. Does the Pentair equipment even support a remote reboot command?

Thanks guys.

**Comment by rstrouse** [comment]
For that OCP there is not a remote reboot command over rs485 that we have seen.  Have you tried to factory reset the OCP and set it back up again?

**Reply by CodeSlinger69** [reply]
Bummer, but thanks for confirming the lack of a remote reboot. I've not tried a factory reset, but I'll give that a try tomorrow and report back.

**Comment by CodeSlinger69** [comment]
Well, I reset my OCP from the Diagnostics menu, and I'm still seeing the same behavior. Periodically, I lose connection, messages get deleted from the buffer because they are not accepted by the OCP, the warning that my RS485 is not connected shows up in dashPanel.

So a bit more thinking made me consider the Elfin EW11 Wifi-RS485 bridge that I am using. I live in the Phoenix metro area, and that OCP box gets hot, even though its shaded from direct sun. I installed the EW11 last fall/winter, so the heat wasn't so bad, but now in June with daily highs in the 100+ range every day, maybe heat is a problem. I had stuffed the EW11 in the wiring section on the left side of the OCP, so it wasn't getting much air circulation. So I grabbed it (and it was hot!), and moved it to a place where more air can circulate around it.

I'll run like that for a while and see if it stays stable.

**Comment by CodeSlinger69** [comment, marked as answer]
So far this has been stable. Looks like it was a heat issue with the EW11, so I'll close this for now. And, in case you are interested, this is moot anyway, since I am starting my Nixie build, which is documented [here](https://github.com/tagyoureit/nodejs-poolController/discussions/801).
